### ISSUE TITLE
Challenges: Triaging Issues

### ISSUE THREAD

**Issue by mgifford** [w3c/wcag#1172]
I do think it is important to triage issues to see that we have the biggest gains in accessibility. 

The User Impact analysis in axe rules are useful.

Useful to have a breakdown like https://accessibility.18f.gov/checklist/

But I also like ideas pioneered by Lucy Greco (to my knowledge) of starting with popular content, content most relevant to PwD, emergency/official content and content requested by PwD. This is useful for videos or PDFs. 

I would like to add that bugs in common templates (header/footer) should be prioritized as they are used everywhere. Beyond that, bugs in common libraries (say in React, CKEditor or Drupal) should take priority, as fixing issues upstream multiplies its benefit. 

There will always be limited resources, so how do we make sure it has the biggest impact. It would be great if WCAG supported guidance for this.

**Comment by mbgower** [comment]
@mgifford I have put a wcag-next label on this, as this is nothing we can address with the existing WCAG 2.x structure.

If you are involved in the WCAG 3 work and feel this is now part of considerations there, feel free to close this. Otherwise, I can escalate if you feel that is not getting sufficient traction.
